src/locales/en/translation.json

```diff
@@ -500,7 +500,8 @@
     },
     "securityCodeWaitToRequest": {
       "title": "You cannot get a new security code at the moment ",
```

Comment:
Is it possible the page is going to have a different `<title>` to the `<h1>` here? For example, might the title be "You cannot get a new security code at the moment" and the heading to be "You cannot sign in at the moment". I'm wondering if we might need to reflect the different headings in the titles too?

Reply:
Nice spot, didn't notice this bug 👍